### PR TITLE
Fixed command for helloworld sample

### PR DESCRIPTION
**Pull request by jatanloya**
Clean PR for previously close PR: https://github.com/openenclave/openenclave/pull/4671
@mingweishih

### CODE REVIEW

**Review by mingweishih — APPROVED** [review]

**Review by anakrish — APPROVED** [review]

File: samples/helloworld/host/Makefile [hunk 1]
```diff
@@ -17,4 +17,4 @@ build:
 	$(CC) -o helloworldhost helloworld_u.o host.o $(LDFLAGS)
```
Comment (mingweishih) [inline samples/helloworld/host/Makefile:17]: Change of `helloworldhost` is missing here

**Review by mingweishih — CHANGES_REQUESTED** [review]

File: samples/helloworld/host/Makefile (hunk 1 above)
Comment (jatanloya) [reply samples/helloworld/host/Makefile]: Fixed

**Review by mingweishih — APPROVED** [review]

**Pull request merged**